api/src/utils/db-constant-utils.ts

```diff
@@ -0,0 +1,25 @@
+import { IDBConnection } from '../database/db';
+import { HTTP400 } from '../errors/CustomError';
+import { getDbCharacterSystemMetaDataConstantSQL } from '../queries/codes/db-constant-queries';
+
+/**
+ * Get db character metadata constants.
+ *
+ * @param {string} constantName
+ * @param {IDBConnection} connection
+ * @return {*} {Promise<void>}
+ */
+export const getDbCharacterSystemMetaDataConstant = async (
+  constantName: string,
+  connection: IDBConnection
+): Promise<any> => {
```

Comment:
nit: this returns either a string (or number?) or null;

Something like: `Promise<string | null>`

Same for the JSDoc on line 10.

Reply:
adjusted